examples/patterns/grid/empty-columns.html

```diff
@@ -16,16 +47,15 @@
       <div class="col-7">
         .col-7
       </div>
-      <div class="col-4 prefix-1">
-        .col-4.prefix-1
+      <div class="col-4 col-start-large-8">
```

Comment:
I don't understand how `col-start-8` would result in the col-4 being pushed over one column. Or is this just a new example?

Reply:
Good spot, I actually hadn't left an empty column there. Fixed. it should start at 9, not 8.

As I understand it, the css grid layout doesn't provide a direct equivalent to the prefixes/sufixes used by the shelves grid. So instead, for each div, you specify start/end position, or one of them together with the number of columns you want it to span. So to achieve the same effect, the div needs to replace ```col-4 prefix-1 ``` with ```col-3 col-start-large-8```.

Reply:
Actually, to achieve the effect of a background wrappping the empty column, we need .col4 and inside, .row>.col-3.col-start-2:
![image](https://user-images.githubusercontent.com/2741678/56301637-221a8a80-6130-11e9-9c2d-afbe0e675de7.png)
I've updated the example.